### PR TITLE
Add contributions and contributors for 1.7.8 RC 1 build 3

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Add contributions and contributors for 178-rc1-build2.
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| How to test?      | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25476)

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -438,6 +438,8 @@ GitHub contributors:
 - Lucas CERDAN
 - Lucas Rolff
 - LucasC
+- Lukas
+- Lukas Laurinaitis
```
Comment (eternoendless) [inline CONTRIBUTORS.md]: Not the same person?

Comment (sowbiba) [reply CONTRIBUTORS.md]: regarding the emails, I have no clue it's the same person :D

Comment (matthieu-rolland) [reply CONTRIBUTORS.md]: it looks like he used his personal and his professional email

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by eternoendless — DISMISSED** [review]
Needs to be rebased and redone on the 1.7.8.0-build branch

**Review by PierreRambaud — APPROVED** [review]

**Comment by eternoendless** [comment]
Build 3 is based on d0972fc

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Pull request merged**